scripts/globals/weaponskills/blade_kamu.lua

```diff
@@ -28,11 +28,11 @@ weaponskill_object.onUseWeaponSkill = function(player, target, wsID, tp, primary
     params.crit100 = 0.0 params.crit200 = 0.0 params.crit300 = 0.0
     params.canCrit = false
     params.acc100 = 0.0 params.acc200 = 0.0 params.acc300 = 0.0
-    params.atk100 = 1; params.atk200 = 1; params.atk300 = 1
+    params.atk100 = 1.3; params.atk200 = 1.3; params.atk300 = 1.3
 
     if xi.settings.USE_ADOULIN_WEAPON_SKILL_CHANGES then
         params.str_wsc = 0.6 params.int_wsc = 0.6
-        params.atk100 = 1.3125; params.atk200 = 1.3125; params.atk300 = 1.3125
+        params.atk100 = 3; params.atk200 = 3; params.atk300 = 3 -- http://wiki.ffo.jp/html/15893.html
```

Comment:
The JP wiki claims it's def ignore +25% attack bonus +125%.  Can you adjust the attack bonuses to match, and add the following?

``` 
params.ignoresDef = true
params.ignored100 = 0.25
params.ignored200 = 0.25
params.ignored300 = 0.25
```

Reply:
I had to triple check before I finally saw the note at the bottom good catch